sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -786,27 +787,8 @@ class ContainerAPITest extends APISpec {
         response.getStatusCode() == 404
     }
 
-    def "Delete if exists service container min"() {
-        setup:
-        def containerName = generateContainerName()
-        primaryBlobServiceClient.createBlobContainer(containerName)
-
-        when:
-        def result = premiumBlobServiceClient.deleteBlobContainerIfExists(containerName)
-
-        then:
-        !premiumBlobServiceClient.getBlobContainerClient(containerName).exists()
-        result
-    }
-
-    def "Delete if exists service container that does not exist"() {
-        when:
-        def response = premiumBlobServiceClient.deleteBlobContainerIfExistsWithResponse(generateContainerName(), null)
-
-        then:
-        response.getStatusCode() == 404
-    }
-
```

Comment:
did these overlap with other tests?

Reply:
No I just didn't like them cus they were failing.

Kidding ;). Yes. There are identical tests in ServiceApiTests, which is where they belong as they are methods on the service client